lib/resources/dstu2/condition.yaml

```diff
@@ -129,22 +129,35 @@ fields:
       "category": {
         "coding": [
           {
-            "system": "http://argonaut.hl7.org/extension-codes",
+            "system": "http://argonaut.hl7.org",
             "code": "problem",
             "display": "Problem"
           }
         ],
         "text": "Problem"
       }
     }
+    OR
+    {
```

Comment:
If the health-concern category is not supported for writes, should we be calling it out here?